package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "onfido-sdk-ui",
-  "version": "3.0.1",
+  "version": "3.1.0-rc.4",
```

Comment:
should it still be rc4?

Reply:
Oh I can see that the push of my latest commit failed. I will push it on Monday from my other laptop